Bridge~/package.json

```
@@ -20,7 +20,7 @@
   "author": "",
   "license": "Apache-2.0",
   "dependencies": {
-    "livekit-client": "^1.15.13",
+    "livekit-client": " v2.5.0",
```

this looks like it's in the wrong format?
```suggestion
    "livekit-client": "^2.5.0",
```